Tests/ComposableArchitectureTests/TestStoreFailureTests.swift

```diff
@@ -266,7 +266,7 @@
     func testExpectedStateEqualityMustModify() async {
       let reducer = Reduce<Int, Bool> { state, action in
         switch action {
-        case true: return Effect(value: false)
+        case true: return EffectPublisher(value: false)
```

Comment:
```suggestion
        case true: return EffectTask(value: false)
```
As this value is usually formed within a `reduce` function (or in a `Reduce` block), it should technically be `EffectTask`.